packages/editor-ui/src/components/Node/NodeCreator/viewsData.ts

```diff
@@ -185,6 +188,15 @@ export function AINodesView(_nodes: SimplifiedNodeType[]): NodeView {
 		};
 	}
 
+	function getSubcategoryInfo(subcategory: string) {
+		const localeKey = `nodeCreator.aiPanel.${camelCase(subcategory)}` as BaseTextKey;
+		if (i18n.exists(localeKey)) {
```

Comment:
TIL this `.exists`, nice! :D

Reply:
TIL that `.exists` does not work with locale keys containing dots =/ Swapped to another check